articles/bastion/quickstart-host-portal.md

```diff
@@ -41,7 +41,8 @@ The following steps walk you through how to deploy Bastion from your VM resource
 
 > [!NOTE]
 > The use of Azure Bastion with Azure Private DNS Zones is not supported at this time. Before you begin, please make sure that the virtual network where you plan to deploy your Bastion resource is not linked to a private DNS zone.
->
+>[!NOTE]
+> West Germany is not currently available for Bastion deployments due to capacity restraints. Please deploy your Bastion resource in another region.
```

Comment:
```suggestion
>
> Also, Germany West Central is not currently available for Bastion deployments due to capacity restraints. Please deploy your Bastion resource in another region.
```

There is no zone called 'West Germany" ... I changed it but need @cherylmc to confirm.

Also,  2 back-to-back 'notes' aren't recommended. combining